recipe/patches/0007-mark-linkage-of-c-ares-re2-zlib-as-private.patch

```diff
@@ -111,30 +111,30 @@ index 1586c0d7fa..0b308c6606 100644
 +  PUBLIC  ${_gRPC_ALLTARGETS_LIBRARIES}
 +  # vendored libs
 +  PUBLIC  ${_gRPC_ADDRESS_SORTING_LIBRARIES}
-+  PUBLIC  upb_message_lib
-+  PUBLIC  utf8_range_lib
++  PUBLIC upb_mini_descriptor_lib
++  PUBLIC upb_wire_lib
 +  # external dependencies used within grpc
-+  PRIVATE ${_gRPC_CARES_LIBRARIES}
 +  PRIVATE ${_gRPC_ZLIB_LIBRARIES}
++  PRIVATE ${_gRPC_CARES_LIBRARIES}
 +  # external dependencies that need to available at runtime
-+  PUBLIC  absl::algorithm_container
-+  PUBLIC  absl::config
-+  PUBLIC  absl::no_destructor
-+  PUBLIC  absl::cleanup
-+  PUBLIC  absl::flat_hash_map
-+  PUBLIC  absl::flat_hash_set
-+  PUBLIC  absl::inlined_vector
-+  PUBLIC  absl::bind_front
-+  PUBLIC  absl::function_ref
-+  PUBLIC  absl::hash
-+  PUBLIC  absl::type_traits
-+  PUBLIC  absl::random_bit_gen_ref
-+  PUBLIC  absl::random_distributions
-+  PUBLIC  absl::statusor
-+  PUBLIC  absl::span
-+  PUBLIC  absl::utility
++  PUBLIC absl::algorithm_container
++  PUBLIC absl::config
```

Comment:
I tried to keep the indentation of `PUBLIC` and `PRIVATE` the same; would appreciate it if you could do that too. It would also cut down heavily on the diff in this patch.

Reply:
This wasn't obivous. I had to fix a conflict here; will fix it.